sample-bt

```diff
@@ -71,11 +71,11 @@ if ($^O ne 'linux') {
 
 my $exec_file = "/proc/$pid/exe";
 if (!-f $exec_file) {
-    die "Nginx process $pid is not running or ",
+    die "Process $pid is not running or ",
         "you do not have enough permissions.\n";
-}
```

Comment:
@secwang I think you should at least run the modified script yourself before committing ;)